### ISSUE TITLE
Support for map format version 4

### ISSUE THREAD

**Issue by FriedrichFroebel** [biylda/WhereYouGo#33]
I recently tried to use offline maps with the WhereYouGo app which were downloaded from [here](http://download.freizeitkarte-osm.de/android/latest/). I used the map files from the 19.06 Freizeitkarte release. But WhereYouGo version 0.9.3 does not accept these maps - I get an error box stating that the map has version 4, but the app supports version 3 only.

I got it to work using the Mapsforge files then (downloaded from [here](http://download.mapsforge.org/maps/v3/)), but they are about a year old now and do not seem to receive updates anymore. [OpenAndroMaps](https://www.openandromaps.org/en/last-update-for-the-old-v3-maps) is soon going to retire the v3 format as well.

Are there any plans to add support for v4 format maps? (Locus is not really an alternative as it requires an additional app to be installed.)

**Comment by ppete2** [comment]
Yes, it would be really important to support offline Mapsorge Maps version 4 and also the associated actual themes. As Friedrich wrote, from September 2019 on the v3 maps of OpenAndroMaps aren't updated anymore, just the v4 ones.

Using online maps instead is not always possible, cause of missing internet connection for example.

**Comment by Lineflyer** [comment]
Repository was relived [here](https://github.com/cgeo/WhereYouGo) here. 
A corresponding issue (see mention link above) has been created in that repository.
